### ISSUE TITLE
DateTime-Locale-1.37 breaks DateTime-1.58

### ISSUE THREAD

**Issue by eserte** [houseabsolute/DateTime-Locale#34]
It seems that DateTime-Locale-1.37 breaks t/13strftime.t in the test suite of DateTime-1.58:
```
    #   Failed test '%c is Sep 7, 1999, 1:02:42 PM'
    #   at t/13strftime.t line 313.
    #          got: 'Sep 7, 1999, 1:02:42 PM'
    #     expected: 'Sep 7, 1999, 1:02:42 PM'

    #   Failed test '%X is 1:02:42 PM'
    #   at t/13strftime.t line 313.
    #          got: '1:02:42 PM'
    #     expected: '1:02:42 PM'
    # Looks like you failed 2 tests of 49.

#   Failed test 'en-US'
#   at t/13strftime.t line 318.
# Looks like you failed 1 test of 11.
t/13strftime.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/11 subtests 
```
The difference seems to be the usage of NARROW NO-BREAK SPACE now.

**Comment by autarch** [comment]
Thanks for the quick report! I'll have a fix out today, I think.

**Comment by autarch** [comment]
Fixed in v1.59
